app/assets/src/components/visualizations/TidyTree.js

```diff
@@ -171,6 +179,31 @@ export default class TidyTree {
     });
   }
 
+  persistCollapsedInUrl(node) {
```

Comment:
Are we going to run into URL limits? There are a lot of nodes on this tree.

Reply:
In the worst case, yes. I intentionally shortened the param names and values. I estimate we can store ~200 collapsed nodes in the URL.  I think it is acceptable to fail after that.